tools/gn

```diff
@@ -579,6 +579,11 @@ def to_gn_args(args):
     if get_host_os() == 'win':
       gn_args['angle_force_context_check_every_call'] = True
 
+    # ANGLE and SwiftShader share build flags to enable X11 and Wayland,
+    # but we only need these enabled for SwiftShader.
+    gn_args['angle_use_x11'] = False
+    gn_args['angle_use_wayland'] = False
```

Comment:
Our tests depend on the Xcb and Wayland Vulkan extensions that are provided by SwiftShader. The latest version of SwiftShader now requires you to opt-in to these extensions through build flags. These same build flags also enable stuff in ANGLE that we don't need.